test/integration/integration.express.test.js

```diff
@@ -14,94 +14,69 @@ const sampleTextReview = fs.readFileSync(filePathReview, 'utf-8');
 const filePathEmail = path.join(__dirname, '../..', '/public/data/personal-email.txt');
 const sampleTextEmail = fs.readFileSync(filePathEmail, 'utf-8');
 
-const describeIfSkip = process.env.TONE_ANALYZER_IAM_APIKEY || process.env.TONE_ANALYZER_USERNAME ? describe : describe.skip;
+const describeIfSkip =
```

Comment:
This seems like strange style to have a line break after an assignment. I'd rather see the first term of the conditional on the same line as the assignment (17), and then the second term on the break.